ui/src/kapacitor/components/KapacitorForm.js

```diff
@@ -154,6 +161,7 @@ KapacitorForm.propTypes = {
   source: shape({}).isRequired,
   addFlashMessage: func.isRequired,
   exists: bool.isRequired,
+  locationState: shape({}),
```

Comment:
could you specify the definition of this prop's shape? it helps other developers know what contract & data shape they can expect when working within this component.

Reply:
changed location state to location and did this.